### PR TITLE
rest of group comparison enrichments tabs

### PR DESCRIPTION
**Pull request by adamabeshouse**

### CODE REVIEW

File: src/pages/resultsView/enrichments/ExpressionEnrichmentsContainer.tsx [hunk 1]
```diff
@@ -40,14 +47,18 @@ export default class ExpressionEnrichmentContainer extends React.Component<IExpr
...
+        return getExpressionRowData(this.props.data, this.props.store ? this.props.store.hugoGeneSymbols : []);
     }
 
     @computed get excludeGenes():string[]|null {
-        if (this.props.store.selectedMolecularProfiles.isComplete &&
-            this.props.store.selectedMolecularProfiles.result
-                .findIndex(x=>x.molecularProfileId === this.props.selectedProfile.molecularProfileId) > -1) {
-            return this.props.store.hugoGeneSymbols;
+        if (this.props.store) {
+            if (this.props.store.selectedMolecularProfiles.isComplete &&
```
Comment (kalletlak) [inline src/pages/resultsView/enrichments/ExpressionEnrichmentsContainer.tsx]: may be combine both if conditions

Comment (adamabeshouse) [reply src/pages/resultsView/enrichments/ExpressionEnrichmentsContainer.tsx]: good catch. ill do the same in AlterationEnrichmentContainer

File: env/group-comparison.sh [hunk 2]
```diff
@@ -0,0 +1 @@
+./rc.sh
```
Comment (alisman) [inline env/group-comparison.sh]: probably shouldn't be committed, right?  you can use custom btw

File: src/pages/groupComparison/CopyNumberEnrichments.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    private onChangeProfile(m:MolecularProfile) {
+        this.props.store.copyNumberEnrichmentProfile = m;
+    }
+
+    readonly tabUI = MakeMobxView({
+        await:()=>[
+            this.props.store.copyNumberHomdelEnrichmentData,
+            this.props.store.copyNumberAmpEnrichmentData
+        ],
+        forcePending:()=>!this.props.store.copyNumberEnrichmentProfile,
```
Comment (alisman) [inline src/pages/groupComparison/CopyNumberEnrichments.tsx]: what is forcePending about?

Comment (adamabeshouse) [reply src/pages/groupComparison/CopyNumberEnrichments.tsx]: its like await, but with any predicate. forcePending might not be the best name.

Comment (alisman) [reply src/pages/groupComparison/CopyNumberEnrichments.tsx]: but what's the basic idea?   like, don't render this if there is no profile?

Comment (adamabeshouse) [reply src/pages/groupComparison/CopyNumberEnrichments.tsx]: yeah, consider it pending if theres no profile

File: src/pages/groupComparison/GroupComparisonStore.ts [hunk 4]
```diff
@@ -113,28 +123,169 @@ export default class GroupComparisonStore {
...
+            if (!this._mutationEnrichmentProfile && this.mutationEnrichmentProfiles.result!.length > 0) {
+                return Promise.resolve(this.mutationEnrichmentProfiles.result![0]);
+            } else {
+                return Promise.resolve(this._mutationEnrichmentProfile);
+            }
         }
-    }
-    public set mutationEnrichmentProfile(profile:MolecularProfile|undefined) {
+    });
+    public setMutationEnrichmentProfile(profile:MolecularProfile|undefined) {
```
Comment (alisman) [inline src/pages/groupComparison/GroupComparisonStore.ts]: @action?   i think it's good to keep up the convention even if when there's only one observable set it means nothing

**Pull request merged**